Logger/FileLogger.cs

```diff
@@ -0,0 +1,22 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Text;
+
+namespace Logger;
+
+    public class FileLogger : BaseLogger
+    {
+        private string _filePath { get; }
```

Comment:
```suggestion
        private string _FilePath { get; }
```
Good job making this readonly!